packages/lib/src/interactions/box.ts

```diff
@@ -32,18 +32,24 @@ class Box extends Box3 {
   }
 
   public expandBySize(width: number, height: number): this {
-    return this.expandByVector(new Vector3(width / 2, height / 2, 0));
+    const { size } = this;
+
+    // Prevent shrinking to "negative" size - i.e. make sure `this.isEmpty()` is never `true`
```

Comment:
Why `isEmpty` is relevant here ?

Reply:
I've improved the comment, you're right :100: -- a better approach would perhaps be to fix `expandByVector`, but overkill for now.